### PR TITLE
fix a few typos in specs

### PR DESCRIPTION
**Pull request by contrun**

### CODE REVIEW

File: specification/src/copy-constraints.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ To establish that the product is correctly computed, the Prover needs to supply
 
 **Protocol Permutation**
  - Verifier is already in possession of polynomial oracles $[t(X)]$ and $[s(X)]$ that agree with $T$ and $S$ on a subgroup $H$ of order $2^k$.
- - Verifier samples $\beta \xleftarrow{\$} \mathbb{F}$ and sends it to Prover.
+ - Verifier samples \(\beta \xleftarrow{\$} \mathbb{F}\) and sends it to Prover.
  - Prover computes running product polynomials, $t_ p(X)$ and $s_ p(X)$ of degree at most $|H|-1$ and sends them to Verifier.
  - Prover and Verifier run an AIR sub-protocol to establish that the AIR is satisfied for $(t(X), s(X), t_ p(X), s_ p(X))$.
 
-What if $T$ and $S$ have multiple columns, say $r$-many? In this case the verifier simply supplies $r$ weights $\alpha_0, \ldots, \alpha_{r-1} \xleftarrow{\$} \mathbb{F}$. The protocol is then applied to the weighted sum of columns.
+What if $T$ and $S$ have multiple columns, say $r$-many? In this case the verifier simply supplies $r$ weights \(\alpha_0, \ldots, \alpha_{r-1} \xleftarrow{\$} \mathbb{F}\). The protocol is then applied to the weighted sum of columns.
```
Comment (sshine) [inline specification/src/copy-constraints.md:28]: These are not typos. The spec consistently uses `$` as the inline MathJax symbol.

**Review by sshine — COMMENTED** [review]
Thank you for noticing the wrong hash length here.

File: specification/src/copy-constraints.md (hunk 1 above)
Comment (contrun) [reply specification/src/copy-constraints.md:28]: Yeah. But it is not rendering correctly in https://triton-vm.org/spec/copy-constraints.html. I wonder what is wrong.

Comment (sshine) [reply specification/src/copy-constraints.md:28]: I'll have a look. Thank you for noticing.

**Review by sshine — APPROVED** [review]
@contrun: It turns out to be a KaTeX bug: While `\$` is the right way to escape a dollar sign, somehow KaTeX captures this and produces a literal `$` which then messes up the inline comment. The `\(`s don't seem to fix it, but adding another backslash does. In LaTeX this would mean "linebreak followed by beginning of inline equation", which would be totally wrong.

**Comment by sshine** [comment]
@jan-ferdinand, @aszepieniec: Beware that `$` is a dodgy character when writing KaTeX.

**Pull request merged**